pyproject.toml

```diff
@@ -95,9 +95,9 @@ numpy = "^1.25.2"
 datasketch = "^1.6.0"
 PyStemmer = "^2.0.1"
 factory-boy = "^3.3.0"
-django-elasticsearch-dsl = "^7.3"
+django-elasticsearch-dsl = "^8.0"
```

Comment:
Changelog: https://github.com/django-es/django-elasticsearch-dsl/blob/master/HISTORY.rst

Reply:
Hm, that changelog is three years old, and I think the reason is that the elastic search DSL library version is supposed to correspond to the version of ES that you're using alongside it, so I think to update this, we'd have to update ES itself.

Reply:
Yeah, elasticsearch update was pulled in at the same time since it's a transient I guess. Here's the changelog for that: https://www.elastic.co/guide/en/elasticsearch/client/python-api/master/release-notes.html

Reply:
Oh, but I think all of this corresponds to the actual version of ES you're running. We should update that, but I doubt you want to wait for that to do this upgrade?